src/backend/distributed/sql/udfs/worker_save_query_explain_analyze/latest.sql

```diff
@@ -0,0 +1,11 @@
+
+CREATE OR REPLACE FUNCTION pg_catalog.worker_save_query_explain_analyze(
+      save_enabled boolean, verbose boolean, costs boolean,
+      timing boolean, summary boolean, query boolean,
+      format integer)
```

Comment:
The PostgreSQL syntax might occasionally be expanded. How would this UDF evolve over time? (considering partial heterogeneous clusters)